website/util/rubeus.py

```diff
@@ -231,10 +207,29 @@ def _serialize_node(self, node, visited=None, parent=None):
             'children': children,
             'isPointer': is_pointer,
             'isSmartFolder': False,
-            'nodeType': node.project_or_component,
-            'nodeID': node.resolve()._id,
+            'nodeType': 'component' if parent else 'project',
+            'nodeID': node._id,
         }
 
+    def _get_nodes(self, node):
+        AbstractNode = apps.get_model('osf.AbstractNode')
+        Contributor = apps.get_model('osf.Contributor')
+        NodeRelation = apps.get_model('osf.NodeRelation')
+
+        data = []
+        if node.can_view(auth=self.auth):
+            serialized_addons = self._collect_addons(node)
+            linked_node_sqs = NodeRelation.objects.filter(parent=node, is_node_link=True)
```

Comment:
Minor: you can use the `related_name` here : `node.node_relations.filter(is_node_link=True)`.